source/include/cfg/cs_Boards.h

```diff
@@ -86,6 +86,8 @@ extern "C" {
 // Crownstone Plug One (first prototype of second edition of the plug)
 #define ACR01B11A            1505
 
+// Outlets
+#define CR01R02v4            1506
```

Comment:
New product type, so lets start a different range.

```suggestion
#define CR01R02v4            1100
```